docs/cni/kubernetes/AWS.md

```diff
@@ -0,0 +1,254 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.14.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Deploying Calico and Kubernetes on AWS 
+
+These instructions allow you to set up a Kubernetes v1.1.4 cluster with Calico networking on AWS using the [Calico CNI plugin][calico-cni]. This guide does not setup TLS between Kubernetes components or on the Kubernetes API.
```

Comment:
It's dangerous putting the exact kubernetes version in here. We'll forget to update it